index.bs

```diff
@@ -417,7 +423,7 @@ An expression is a series of [=sato/tokens=]. A [=sato/token=] is an [=sato/oper
 aligned(8) class SampleTransform {
     unsigned int(2) version = 0;
     unsigned int(4) reserved;
-    unsigned int(2) bit_depth; // Signed 8, 16, 32 or 64-bit.
+    unsigned int(2) bit_depth; // Enum signaling 8, 16, 32 or 64-bit.
```

Comment:
"Signed" was intended, as in not unsigned

Reply:
Good point. Will change to say it's an enum signaling xxxx